.github/workflows/check-patch.yml

```diff
@@ -33,10 +33,19 @@ jobs:
 
     steps:
       - name: prepare env
+        if: ${{ matrix.shortcut == 'cs9' }}
         run: |
             dnf install -y --setopt=tsflags=nodocs git rpmlint glibc-langpack-en
 
-      - uses: actions/checkout@v2
+      - name: prepare env
+        if: ${{ matrix.shortcut == 'cs10' }}
+        run: |
+          dnf install https://dl.fedoraproject.org/pub/epel/epel-release-latest-10.noarch.rpm
```

Comment:
isn't it enough to just call `dnf install -y epel-release` ?

Reply:
https://docs.fedoraproject.org/en-US/epel/getting-started/
epel-release doesn't seem to be in CS10 repos

Reply:
It's in extras-common which should be enabled by default: https://mirror.stream.centos.org/SIGs/10-stream/extras/x86_64/extras-common/Packages/e/

Reply:
Missed that, fixed